bit_manipulation/bitmask.cpp

```diff
@@ -0,0 +1,167 @@
+/**
+ * @file
+ * @brief Bitmask example
```

Comment:
Please provide a better brief description.
Also, I think the following link is actually better (please do check out if I'm correct).
```suggestion
 * @brief [Bitmask](https://en.wikipedia.org/wiki/Mask_(computing)) example
```